File: Employee Directory/ClientApp/src/app/models/JobTitle.ts
export class JobTitle {
  public id: number;
  public name: string;
  public departmentId: number;

  constructor(args: any) {
    args = !!args ? args : {};
    this.id = args.id;
    this.name = args.name;
    this.departmentId = args.departmentId;
  }
}

File: Employee_Directory.Models/JobTitle.cs
using PetaPoco;
using System;
using System.ComponentModel.DataAnnotations;

namespace EmployeeDirectory.DataModels
{
    [TableName("JobTitles")]
    [PrimaryKey("Id")]
    public class JobTitle
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public int DepartmentId { get; set; }

        public DateTime CreatedOn { get; set; }

        public string CreatedBy { get; set; }

        public DateTime ModifiedOn { get; set; }

        public string ModifiedBy { get; set; }
    }
}
